ckanext/dcat/converters.py

```diff
@@ -93,9 +104,16 @@ def ckan_to_dcat(package_dict):
 
     dcat_dict['distribution'] = []
     for resource in package_dict.get('resources', []):
+        context = {u'model': model, u'session': model.Session,
+                   u'user': g.user, u'auth_user_obj': g.userobj}
+        try:
+            res_dict = get_action(u'resource_show')(context, {'id': resource.get('id')})
```

Comment:
Why do we need to call `resource_show`  here?

Reply:
Sorry I was bringing code over from a different fork, and that was on an old ckan version. Tested without the `resource_show` and it works as expected.